tests/cms_instance_test.php

```diff
@@ -59,9 +61,12 @@ public function test_add_instance() {
         $moduleinfo = $this->create_module($cmstype->get('id'), $course->id);
 
         // Check that the instance and datasource info exists.
-        $instanceid = $moduleinfo->instance;
+        $cmsid = $moduleinfo->instance;
+
+        $count = $DB->count_records('cms', ['id' => $cmsid]);
```

Comment:
Is there a reason we're not using the cms persistent for this? `cms::count_records(['id' => $cmsid]);`

Reply:
done